main/src/vtr_lidar/src/path_planning/cbit.cpp

```diff
@@ -296,14 +304,14 @@ auto LidarCBIT::computeCommand(RobotState& robot_state0) -> Command {
     w_p_r_in_r(3) = 0.0;
     w_p_r_in_r(4) = 0.0;
     w_p_r_in_r(5) = -1* vel_history.back()[1];
-    CLOG(DEBUG, "mpc_debug.cbit") << "Robot velocity Used for Extrapolation: " << -w_p_r_in_r.transpose() << std::endl;
+    CLOG(ERROR, "mpc_debug.cbit") << "Robot velocity Used for Extrapolation: " << -w_p_r_in_r.transpose() << std::endl;
```

Comment:
These don't look like errors, I think this should be reverted to DEBUG.

Reply:
correct, missed these in my cleanup - resolved in latest commit